File: ShopServer/ShopServer.Entity/ShoppingCar.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ShopServer.Entity
{
    public class ShoppingCar
    {
        public int Id { get; set; }
        public int CustomerId {  get; set; }
        public Customer Customer { get; set; }
        public int ItemId { get; set; }
        public Item Item { get;set; }
        public DateTime Date { get; set; }
        public int Count { get; set; }
    }
}

File: ShopServer/Shop/src/app/models/ShoppingCar.ts
import { Customer } from "./customer";
import { Item } from "./Item";

export class ShoppingCar{
    id :number=0;
    customerId:number=0; 
    customer :Customer=new Customer;
    itemId :number=0;
    item :Item=new Item;
    date :Date=new Date;
    count :number=0;

}